src/main/webapp/dev/js/common/bootboxWrapper.js

```diff
@@ -47,13 +47,13 @@ function showModalAlert(titleText, messageText, okButtonText, bootstrapContextua
  * Custom confirmation dialog to replace default confirm() function
  * Required params: titleText, messageText and okCallback
  * Optional params: cancelCallBack (defaults to null)
- *                  okButtonText (defaults to "OK")
```

Comment:
I don't think this is the correct to make the changes. 

By modifying this, you are saying that by default, all `yesButtonText` is `Yes` rather than `Ok` (which is not always the case and the issue only says to modify modal related to sessions).